test/new-e2e/tests/agent-shared-components/config-refresh/non_core_agents_sync_test.go

```diff
@@ -88,6 +89,12 @@ func (v *configRefreshSuite) TestConfigRefresh() {
 			agentparams.WithSecurityAgentConfig(securityAgentConfig),
 			agentparams.WithSkipAPIKeyInConfig(), // api_key is already provided in the config
 		),
+		awshost.WithAgentClientOptions(
+			agentclientparams.WithAuthTokenPath(authTokenFilePath),
+			agentclientparams.WithTraceAgentOnPort(apmCmdPort),
+			agentclientparams.WithProcessAgentOnPort(processCmdPort),
```

Comment:
We can use 
```
func WithProcessAgent() Option {
	return WithProcessAgentOnPort(6162)
}
```
Since `processCmdPort` is already equal to `6162` ?

Reply:
I did it that way in case we change the value of `processCmdPort` (it avoids changing two lines)